### DISCUSSION TITLE
Playlists overrunning and causing the next one to run late

### DISCUSSION THREAD

**Discussion by stevemix** [AzuraCast/AzuraCast#7510] [General]
Hi 
Just wondered if anyone could offer some advise on the following.
When I schedule a playlist if a song is going to run over the scheduled time of that playlist it puts the other playlist late starting. is there a workaround for this please.

Many Thanks
Steve

**Comment by Heimatradio** [comment]
![Unbenannt-2](https://github.com/user-attachments/assets/78c6e129-44a5-4138-b269-3a6a1530d9af)
Set "Interrupt other songs to play at scheduled time" in your playlists

**Reply by stevemix** [reply]
tried this and it causes the file in the playlist to repeat
